app/app.iml

```diff
@@ -17,17 +17,15 @@
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/gen" isTestSource="false" generated="true" />
     </content>
-    <orderEntry type="jdk" jdkName="Android API 21 Platform" jdkType="Android SDK" />
+    <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="libs" level="project" />
     <orderEntry type="module" module-name="achartengine" />
     <orderEntry type="module" module-name="android-mapviewballoons" />
+    <orderEntry type="module" module-name="android-GridViewWithHeaderAndFooter" />
```

Comment:
Needed for the new ui. If you don't have it then from commcare-odk dir run:
git submodule update --init --recursive